src/js/ui/home.js

```diff
@@ -0,0 +1,26 @@
+import { capitalizeFirstLetter, getID } from '../utils.js';
+
+function createPokemonTile(pokemon, sprite) {
+    const $template = document.getElementById('tile-template').content.cloneNode(true);
+
+    const { name, url } = pokemon;
+    const ID = getID(url);
+    
+    $template.querySelector('p').textContent = capitalizeFirstLetter(name);
+    $template.querySelector('img').src = sprite(ID);
+    $template.querySelector('img').dataset.id = ID;
+    $template.querySelector('div').dataset.id = ID;
+    
+    return $template;
+}
+
+function appendPokemon($pokemon) {
```

Comment:
You can rename this function to `append($pokemon)`. This is more of a suggestion to avoid repeating the pokemon word, like `appendPokemon($pokemon)`